src/mesh/elem2NodalLoads.m

```diff
@@ -104,16 +104,17 @@
       nodes = Conec( elem, 3+(1:3) ) ;
 
       % area of the element with direction
-      areaElem = 0.5 * norm( cross( ...
+      areaElem = 0.5 * cross( ...
```

Comment:
```suggestion
      Normal = cross( ...
```

Reply:
I suggest defining first the normal vector and then computing the area with that and then normalize the vector and get n